ansible_collections/arista/avd/examples/single-dc-l3ls/group_vars/FABRIC.yml

```diff
@@ -49,3 +49,41 @@ bgp_peer_groups:
 # If you're running vEOS-lab or cEOS, you should use MTU of 1500 instead as shown in the following line
 # p2p_uplinks_mtu: 1500
 p2p_uplinks_mtu: 1500
```

Comment:
This is confusing. I think the example should have 9214 or whatever, and then you have your commented 1500.